src/main/java/seedu/duke/component/Resistor.java

```diff
@@ -0,0 +1,19 @@
+package seedu.duke.component;
+
+public class Resistor extends Component {
+    private static final String SI_UNIT = "μH";
```

Comment:
Consider correcting the `SI_UNIT`?
```suggestion
    private static final String SI_UNIT = "Ω";
```